vite.config.ts

```diff
@@ -68,22 +75,30 @@ export default defineConfig({
         "./src/styles/theme.css",
       ],
     },
-    proxy: {
-      // Any new endpoints defined in the backend server need to be added here
-      // as vite serves the frontend during local development and in the live preview,
-      // and needs to know to proxy the endpoints to the backend server.
-      "/_spark/kv": {
-        target: "http://localhost:8000",
+    proxy: {      
+      "^/_spark/kv": {
+        target: GITHUB_API_URL,
         changeOrigin: true,
+        rewrite: (path) => {
+          return path.replace(/^\/_spark\/kv/, `/runtime/${GITHUB_RUNTIME_PERMANENT_NAME}/kv`);
+        },
+        configure: addGitHubAuth
       },
```

Comment:
Does KV have to be separate from the `.*` ones? Thought it was just like the others.

Also I think this is out of date with main since I consolidated the paths when I needed to add the `loaded` route.